Form-Controls/index.html

```diff
@@ -13,15 +13,44 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+          <fieldset>
+            <legend>Confirm Personal Details</legend>
+            <label for="name">First Name</label><br />
+            <input type="text" id="name" name="name" required minlength="2" placeholder="John"><br />
```

Comment:
🤔 I don't know if this placeholder adds much value to the form since you have a descriptive label above it.